.github/CHANGELOG.md

```diff
@@ -1,10 +1,29 @@
 ## Release 0.2.0 (development release)
 
+### New features since last release
+
+### Improvements
+
+* The maximum wire value used in a statement within a definition will be the number of wires
+  declared. E.g., if, within a definition, a single statement is applied on wire 4, then the
+  declared wires will be 0, 1, 2, 3, and 4.
+  [(#2)](https://github.com/XanaduAI/xir/pull/2)
+
+### Bug Fixes
+
+* Wires are implicitly added to the declaration if not explicitly declared in a definition.
```

Comment:
This is regarding observables, right?
```suggestion
* Wires are implicitly added to an observable declaration if they are not explicitly declared.
```

Reply:
I guess the issue was regarding observables specifically, although I did some minor changes to how the gate definition wires are stored as well (although, I think they technically worked before as well).